src/components/number/number.stories.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  text,
+  number,
+  select,
+  boolean
+} from '@storybook/addon-knobs';
+import Number from './number';
+import notes from './notes.md';
+import OptionsHelper from '../../utils/helpers/options-helper';
+
+storiesOf('Number Input', module)
+  .add('default', () => {
+    const inputWidth = text('inputWidth', '');
+    const timeToDisappear = number('timeToDisappear', 0);
+    const fieldHelp = text('fieldHelp', 'This text provides help for the input.');
+    const fieldHelpInline = boolean('fieldHelpInline', false);
+    const label = text('label', 'Example NumberInput');
+    const labelInline = label ? boolean('labelInline', false) : undefined;
+    const labelWidth = labelInline ? text('labelWidth', '') : undefined;
+    const labelAlign = labelInline ? select(
+      'labelAlign',
+      OptionsHelper.alignBinary,
+      OptionsHelper.alignBinary[0]
+    ) : undefined;
+    const labelHelp = text('LabelHelp', 'This text provides more information for the label.');
+
+    return (
```

Comment:
This message appears bunched up agains the top of the window.

![image](https://user-images.githubusercontent.com/26925206/53879348-223e4b00-4006-11e9-9af9-138ff76bbe38.png)

Reply:
Fixed 👍